packages/forms/src/model/form_control.ts

```diff
@@ -309,34 +323,44 @@ export interface ɵFormControlCtor {
    * such functions, or a `FormControlOptions` object that contains validation functions
    * and a validation trigger.
    *
-   * @param asyncValidator A single async validator or array of async validator functions.
+   * @param asyncValidator A single async validator or array of async validator functions
    */
-  new(formState: any, validatorOrOpts?: ValidatorFn|ValidatorFn[]|FormControlOptions|null,
-      asyncValidator?: AsyncValidatorFn|AsyncValidatorFn[]|null): FormControl;
+  new<T = any>(value: FormState<T>|T, opts: FormControlOptions&{initialValueIsDefault: true}):
+      FormControl<T>;
+  new<T = any>(
+      value: FormState<T>|T, validatorOrOpts?: ValidatorFn|ValidatorFn[]|FormControlOptions|null,
+      asyncValidator?: AsyncValidatorFn|AsyncValidatorFn[]|null): FormControl<T|null>;
 
   /**
    * The presence of an explicit `prototype` property provides backwards-compatibility for apps that
    * manually inspect the prototype chain.
    */
-  prototype: FormControl;
+  prototype: FormControl<any>;
+}
+
+function isFormState<T>(formState: unknown): formState is FormState<T> {
```

Comment:
```suggestion
function isFormState(formState: unknown): formState is FormState<unknown> {
```

There's no need for this function to make any assertions about the generic type of the `FormState` - the `<T>` parameters is never passed in this code anyway.

Reply:
Makes sense, fixed.